lib/pointing_party_web/live/card_live.ex

```diff
@@ -0,0 +1,114 @@
+defmodule PointingPartyWeb.CardLive do
+  use Phoenix.LiveView
+
+  alias PointingParty.{Card, VoteCalculator}
+  alias PointingPartyWeb.{Endpoint, Presence}
+
+  def render(assigns) do
+    Phoenix.View.render(PointingPartyWeb.CardView, "index.html", assigns)
+  end
+
+  def mount(%{username: username}, socket) do
+    Endpoint.subscribe("users")
+    {:ok, _} = Presence.track(self(), "users", username, %{points: nil})
+    assigns = [
+      current_card: nil,
+      is_driving: false,
+      outcome: nil,
+      party_has_started: false,
```

Comment:
kind of a nit pick but can we call `party_has_started` `pointing` instead? It fits better on the slides!

Reply:
What about `is_pointing` or `has_started`?

Reply:
Yea either of those sound okay to me!